rasterio/rio/info.py

```diff
@@ -245,7 +245,8 @@ def info(ctx, input, aspect, indent, namespace, meta_member, verbose, bidx,
                     proj4 = proj4.split('=')[1].upper()
                 info['crs'] = proj4
                 info['res'] = src.res
-                info['lnglat'] = src.lnglat()
+                if proj4 != '':
+                    info['lnglat'] = src.lnglat()
```

Comment:
otherwise you will get the exception raised by lnglat(). That said, would perhaps be better to catch the exception instead of doing this prior sanity check